drape_frontend/apply_feature_functors.hpp

```diff
@@ -21,6 +21,11 @@ class SymbolRuleProto;
 
 //#define CALC_FILTERED_POINTS
 
+namespace ftypes
+{
+class RoadShield;
+}  // namespace ftypes
```

Comment:
Note: generally for namespaces in forward-declaration we do not write trailing "// namespace ABC" comment. Feel free to remove it.